project/core/settings.py

```diff
@@ -54,6 +54,7 @@
     "activities",
     "caregivers",
     "people",
+    "password",
```

Comment:
Since account-related features reside in the `accounts` app, we probably don't need a `password` app just to change the user password. Let's add the password reset feature to the `accounts` app and remove the unneeded `password` app.